packages/@lwc/engine-core/src/framework/rendering.ts

```diff
@@ -664,23 +739,13 @@ function createViewModelHook(elm: HTMLElement, vnode: VCustomElement, renderer:
     return vm;
 }
 
-/**
- * Collects all slots into a SlotSet, traversing through VFragment Nodes
- */
 function collectSlots(vm: VM, children: VNodes, cmpSlotsMapping: { [key: string]: VNodes }) {
```

Comment:
Not used anymore. Another suggestion is to move this function back inline. IIRC, this was inline to begin with and was moved out to allow recursion. 
```suggestion
function collectSlots(children: VNodes, cmpSlotsMapping: { [key: string]: VNodes }) {
```